packages/graphql/tests/tck/union.test.ts

```diff
@@ -30,7 +30,7 @@ describe("Cypher Union", () => {
 
     beforeAll(() => {
         typeDefs = gql`
-            union Search = Movie | Genre
+            union Search = Genre | Movie
```

Comment:
Why this change? Does it change the order of the resulting union in cypher?

Reply:
Yes, the order in which the union subqueries are depends on the parsing order in the Schema Model, I tried to find a consistent order logic in the previous implementation but I failed. I make these small changes in the type definitions to make it easier to detect the actual changes!